client/beta/faq.jsx

```diff
@@ -9,44 +8,40 @@ import QuestionIcon from '../icons/material/ic_help_outline_black_48px.svg'
 
 const questions = [
   {
-    question: 'When will the open beta start?',
+    question: 'What version of BW does ShieldBattery support?',
     answer: [
       <p key="p1">
-        Open beta will start once ShieldBattery is stable and ready for the public. An exact ETA for
-        this is unfortunately impossible to give, but expect to receive regular updates from us as
-        we make progress. Make sure to follow our{' '}
-        <a href="https://twitter.com/shieldbatterybw" target="_blank">
-          Twitter account
-        </a>{' '}
-        to keep up with the latest news and changes.
+        ShieldBattery supports all versions of BW from 1.16.1 and up. The version that is actually
+        run when playing on ShieldBattery is 1.16.1. This means that you can give our application an
+        install path to any version of BW 1.16.1 and above, and things will just work.
       </p>,
       <p key="p2">
-        Even before the open beta starts, we'll be inviting new users to our closed beta
-        periodically. So if you're interested in helping us test the software, finding bugs, and
-        giving feedback before it's available to a wider audience, please{' '}
-        <Link to="/splash">sign up</Link> for an invite.
+        BW:Remaster is not out yet, but to play on it using ShieldBattery, you will have to purchase
```

Comment:
I don't think I would guarantee compatibility with StarCraft: Remastered at this point. We don't know how it will be structured at all.

#1) Always refer to it as "StarCraft: Remastered"
#2) Remove any references to it here, point people to the free StarCraft download link if you want to reference anything.

If/when in the future we have support for that version, we can add/adjust this answer then